### PR TITLE
Governance documents for ZLC approval

### PR DESCRIPTION
**Pull request by jmertic**

### CODE REVIEW

File: process/roles.md [hunk 1]
```diff
@@ -33,6 +31,17 @@ Rights and responsibilities of a Committer include, but are not limited to the f
 * Proactively report problems in the bug tracking system, and annotate problem reports with status information, explanations, clarifications, or requests for more information from the submitter. Committers are responsible for updating problem reports when they have done work related to the problem.
 * Be responsive to questions raised about the sub-project on mailing lists and other mediums.
 
-# Lead
+### Process for Becoming a Committer
+
+* Demonstrate your experience with the codebase through contributions and engagement on the community channels.
+* Normally the community is aware of activities by Contributors and ideally a nomination would be made by an existing committer.  However, self nomination is an acceptable means of identifying your interest.  This interest should be expressed in an approrpriate e-mail channel; the only persistent means of communication.  See the  [community repo](https://github.com/zowe/community/) for more details.
```
Comment (1000TurquoisePogs) [inline process/roles.md]: approrpriate  -> appropriate

File: README.md [hunk 2]
```diff
@@ -36,37 +36,41 @@ All ZLC meeting are posted on the [Zowe Development Calendar](https://lists.open
...
-* Zowe CI/CD
+* [Zowe API Mediation layer](https://github.com/zowe/community#zowe-api-mediation-layer)
+* [Zowe CLI](https://github.com/zowe/community#zowe-cli)
+* [Zowe App Framework](https://github.com/zowe/community#zowe-app-framework)
 
 ## Emeritus
 
+* *None*
+
+There are other efforts that sit outside the project structure that focus on cross sub-project coordination. Learn more about the [Zowe Community Teams](https://github.com/zowe/community#zowe-community-teams).
```
Comment (1000TurquoisePogs) [inline README.md]: Because this is put below emeritus instead of active, it may be confusing as to the status. Maybe not, just a thought though.

Comment (jmertic) [reply README.md]: Thanks - moved it up to avoid that confusion.

File: process/roles.md (hunk 1 above)
Comment (jmertic) [reply process/roles.md]: Thanks - fixed misspelling.

**Review by 1000TurquoisePogs — APPROVED** [review]

**Review by jplinardon — APPROVED** [review]

**Pull request merged**